tests/meta/workflow/conftest.py

```diff
@@ -158,20 +193,35 @@ def qc_microsalt_context(
         internal_id=microsalt_case_qc_pass,
         name=microsalt_case_qc_pass,
         data_analysis=Pipeline.MICROSALT,
+        action=CaseActions.RUNNING,
     )
 
-    for sample in qc_pass_microsalt_samples:
+    for sample in qc_pass_microsalt_samples[1:]:
```

Comment:
Why do we skip the first sample?

Reply:
Because it is created below as a negative control